csrankings-s.csv

```diff
@@ -506,7 +506,7 @@ Sebastian Wandelt,Beihang University,http://m3nets.de/group/sw.html,lL0jebkAAAAJ
 Sebastian Wild,University of Liverpool,https://www.liverpool.ac.uk/computer-science/staff/sebastian-wild/publications,aMyZiK0AAAAJ
 Sebastian Zug,TU Freiberg,https://tu-freiberg.de/fakult1/inf/professuren/softwaretechnologie-und-robotik/team,D6A7l-MAAAAJ
 Sebastian von Mammen,University of Würzburg,http://hci.uni-wuerzburg.de/people/sebastian-von-mammen,Rni2mdIAAAAJ
-Sébastien Ourselin,University College London,http://cmic.cs.ucl.ac.uk/staff/sebastien_ourselin,SMvz9eEAAAAJ
+Sébastien Ourselin,King's College London,https://www.kcl.ac.uk/people/sebastien-ourselin,SMvz9eEAAAAJ

```

Comment:
Not clear he continues to meet inclusion criteria (not in CS, no obvious appointment in CS).

Reply:
Yes, it looks like he is not in CS, so should be removed.

Reply:
OK - if you could just remove him in the PR, that'd be much appreciated.